lib/workers/pr/changelog/source-github.ts

```diff
@@ -78,13 +78,14 @@ export async function getChangeLogJSON({
       );
       return { error: ChangeLogError.MissingGithubToken };
     }
+    // This should change now that gitlab is another option for retrieval
```

Comment:
Change how?

Reply:
Sorry, I was wrong when writing that, there's a separate `source-gitlab` which I didn't consider when looking at this debug message mentioning github. Updated.